src/index.ts

```diff
@@ -72,6 +75,7 @@ export default class Api {
     this.timeTrackings = new Timetrackings(this.config);
     this.users = new Users(this.config);
     this.userStatusOverviews = new UserStatusOverviews(this.config);
+    this.generalSettings = new GeneralSettings(this.config);
```

Comment:
Change readme to reflect that general settings are covered (as there is read only).